SmartPlaces.Facilities/README.md

```diff
@@ -1,27 +1,69 @@
+There are currently 4 library projects:
 
-1. Clone the repo
-3. Compile the Solution.
+| Name | Description | Read More |
+| --- | --- | --- |
+| OntologyManager | Defines base classes and helpers which allow mapping from one ontology to another | [Read More](./lib/OntologyMapper/README.md) |
+| OntologyManager.Mapped | Defines the mappings from the Mapped ontology to Rec and Willow ontologies | [Read More](./lib/OntologyMapper.Mapped/README.md) |
+| IngestioManager | Defines the process flow for copying one topology to another topology and using the Ontology Mapper to convert the source ontology to a target ontology | [Read More](./lib/IngestionManager/README.md) |
+| IngestionManager.Mapped | Refines specific functionality needed when using Mapped as an input topology | [Read More](./lib/IngestionManager.Mapped/README.md) |
+
+The nugets can be found on [Nuget.org](https://www.nuget.org/packages?q=Microsoft.SmartPlaces).
+
+### Getting Started with the Lib projects
+
+Each library folder has a stand-alone solution which includes a src and test project.
+
+Dependencies between the projects are taken via Nugets, which can be a little bit painful at times, but we are working to improve that experience.
+
+When updating the src project for any of the libs, please remember to:
+1. Update the Assembly Version and Package Version in the csproj as appropriate. The projects use [Semantic Versioning](https://semver.org/), so please follow the semantic versioning guidelines when updating.
+2. Add Unit tests for any new features you add or bugs you fix
+3. Ensure the documentation for the lib is up to date
+4. Upon completion of the pull request, sent a note to [smartplaces@microsoft.com](mailto:smartplaces@microsoft.com) to request that they publish the latest version of the Nuget packages. Response times should be under 2 business days.
+5. If you are updating a low level project that one of the other projects depend upon through a nuget reference, you will need to do 2 PRs: one for the low level package, and one for the lib referencing the updated package. Note that the second pull request will not succeed until step 4 above is completed by the Smart Places team. 
+
+### Local Testing
+
+Because of the complexity of the process of getting the official Nuget packages updated, it is always faster to test all the components locally before starting the first pull request. Here are some steps to make testing simpler:
+
+1. Set up a local folder on your machine to put Nuget packages into (i.e. c:\nuget)
+2. Update the [Nuget.config](./NuGet.config) and add the following line to it. Never check in this file. It will break the builds.
+
+    ```
+
+    <add key="DELETE_ME" value="c:\nuget" />
+
+    ```
+
+3. Update the version number in the lib you have changed
+4. Run the following command at a command prompt:
+
+    ```
+    
+    dotnet nuget pack <project file>
```

Comment:
Might help tp add the output directory to match the Delete_Me added above

`dotnet pack .\lib\InfrastructureConfiguration\src\ -o ..\LocalNugets`